app/src/androidTest/java/com/github/se/icebreakrr/ui/sections/NotificationTest.kt

```diff
@@ -107,8 +155,48 @@ class NotificationTest {
   }
 
   @Test
-  fun profilesListNotEmpty() {
-    // TODO: add test for profile cards in the Notification Screen when having meeting request
+  fun InboxListOneElement() {
+    val updatedProfile = profile1.copy(meetingRequestInbox = mapOf(("Jean" to "hello !")))
+    `when`(mockProfilesRepository.getProfileByUid(eq("1"), any(), any())).thenAnswer {
+      val onSuccess = it.getArgument<(Profile) -> Unit>(1)
+      onSuccess(updatedProfile)
+    }
+    `when`(mockProfilesRepository.getMultipleProfiles(any(), any(), any())).thenAnswer {
+      val onSuccess = it.getArgument<(List<Profile>) -> Unit>(1)
+      onSuccess(listOf(profile2))
+    }
+    composeTestRule.setContent {
+      NotificationScreen(navigationActions, profilesViewModel, meetingRequestViewModel)
+    }
+    composeTestRule.waitForIdle()
+    composeTestRule.onNodeWithTag("topBar").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("bottomNavigationMenu").assertIsDisplayed()
+    composeTestRule.onNodeWithTag("profileCard").assertIsDisplayed()
```

Comment:
To be even more precise about this element's behavior, you can check if it has a click action.

Reply:
Yes but the problem is that the click action is still undefined I think, well they have to navigate to InboxProfileView, I can add a test to verify if you navigate to this page ?

Reply:
Like check if it has a click action with a performclick ? Yes i can try to add that

Reply:
I meant that you can check if an object has a clickable action, but `performClick()` works well too

_Trick_ : you can stack asserts, that means that you can have something like that : 

`composeTestRule.onNodeWithTag("profileCard").assertIsDisplayed().assertHasClickAction()`